_appmap/test/test_env.py

```diff
@@ -11,3 +12,17 @@ def test_disable_temporarily():
     except RuntimeError:
         ...
     assert env.enables("requests")
+
+def test_determine_log_file(monkeypatch):
+    env = Env({"_APPMAP": "true"})
```

Comment:
Did you check to see if `APPMAP` was really `"true"` when you were investigating #339? I don't think it should have been.